REST/RESTListener+Handlers.cc

```diff
@@ -287,8 +287,11 @@ namespace litecore { namespace REST {
 
         // Encode body as Fleece (and strip _id and _rev):
         alloc_slice encodedBody = c4doc_encodeStrippingOldMetaProperties(body,
-                                                                         c4db_getFLSharedKeys(db));
-
+                                                                         c4db_getFLSharedKeys(db),
+                                                                         outError);
+        if (!encodedBody && outError && outError->domain != 0 && outError->code != 0)
```

Comment:
No need to check `outError`; if `encodedBody` is null, that means the call failed. (And even if the caller passed null for `outError`, this method should still return false.)